projects/plugins/boost/app/features/image-guide/src/ui/Popup.svelte

```diff
@@ -0,0 +1,231 @@
+<script lang="ts">
```

Comment:
Not a strong opinion. Would it benefit from a different name as Popup has a different meaning? Maybe something like `GuideCard`.

Reply:
I was calling this `ImageGuide` previously and changed it to `Popup` because it's when jumping files int he image-guide project it's much easier to remember "I'm looking for a popup" rather than "I want to edit the pop-up called ImageGuide now". Naming things is hard, I think `GuideCard` is also not as useful. I could settle for `GuidePopover`, but, personally, I think `Popup` and `Bubble` components standing next to each other explain themselves pretty well.